core/notifications/src/executor/novu/subscriber.rs

```diff
@@ -234,6 +234,7 @@ impl Subscribers {
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
```[allow(dead_code)]``` is required here. Tried to prefix it as well but then clippy started giving other warnings.